lib/cog/commands/help.ex

```diff
@@ -75,11 +61,84 @@ defmodule Cog.Commands.Help do
       {:ok, body} ->
         {:reply, req.reply_to, body, state}
       {:error, error} ->
-        {:error, req.reply_to, error, state}
+        {:error, req.reply_to, error_msg(error), state}
+    end
+  end
+
+  # If multiple args are passed then we join with "-" before doing a lookup.
+  # This should account for users passing command names using the optional
+  # "git style" syntax.
+  defp lookup(args) when is_list(args) do
+    Enum.join(args, "-")
```

Comment:
@kevsmith thoughts on this? I think it's less complex and, we save both the call to the parser and the extra db query. But as a trade off users could technically do something like, `help test bundle:test command` and get the help for `test-bundle:test-command` whereas the command `test bundle:test command` would fail to parse normally.